bluetti_mqtt/core/devices/ac200m.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, address: str, sn: str):
         self.struct.add_uint_field('pack_num', 0x0B, 0xBE)
         self.struct.add_bool_field('ac_output_on', 0x0B, 0xBF)
         self.struct.add_bool_field('dc_output_on', 0x0B, 0xC0)
+        self.struct.add_bool_field('power_off', 0x0B, 0xF4)
```

Comment:
Can you please move this line down 1 to follow the comment? The 0xF4 offset comes after the 0xD7-0xD9 offsets.